client/a8c-for-agencies/sections/overview/overview.tsx

```diff
@@ -35,6 +41,35 @@ export default function Overview() {
 				</LayoutHeader>
 			</LayoutTop>
 			<LayoutBody className="a4a-overview-content">
+				<Button
+					variant="secondary"
+					onClick={ () =>
+						createSuccessNotice( translate( 'This is a notice with link' ), {
+							id: 'notice-2',
+							actions: [
+								{
+									label: 'Action',
+									url: null,
+									onClick: null,
+								},
+							],
+						} )
+					}
+				>
+					Notice with Action
+				</Button>
+				&nbsp;&nbsp;&nbsp;
```

Comment:
What are the 3 non-breaking spaces for?

Reply:
This is just a POC I did, we are not going to merge this PR. This was to add some spacing between the buttons I added to show the notices.